### DISCUSSION TITLE
RX 580 8GB stops mining Beam?

### DISCUSSION THREAD

**Discussion by Fohsap** [Lolliedieb/lolMiner-releases#368] [Q&A]
'Press any key to continue...'
I get this message after a few minutes of mining.  It just stops mining.  Can someone please explain why this is happening and how to make it stop?

**Comment by MiroslavKatic** [comment]
> 'Press any key to continue...'
> I get this message after a few minutes of mining. It just stops mining. Can someone please explain why this is happening and how to make it stop?

Same here!
2 x RX 580 4GB, random crashes, usually 3-7 in 24h. Sometimes I have luck to mine 8 hours continuously, but sometimes it can crash 2 times in one hour. I was really excited to read following sentence: "_Fixed some (rare) potential faults in Beam stratum_" in change log for 1.26, upgraded to 1.26 from 1.21 instantly, but after 24h I had 4 crashes.

system:
Windows 10 x64, dual mining XMR (CPU) + beam, but one dedicated core for lolMiner (CPU affinity for lolMiner process on that core (in custom powershell "watcher" script, because of crashes)).  3 different Adrenalin drivers, starting from 21.3.x. One one AMD wattman reset to default error in first 10 days, tweaked GCPU/VRAM voltage a little, no wattman issue in last 1.5 months. Last line in log usually: _GPU X: Share accepted (36 ms)_. I didn't try any other algo. Running it as system account under logon screen, not in desktop. RAM (16GB) isn't a issue, because xmrig runs stable. RX 580 running in PCIE slots on motherboard, no riser.

I though I was alone with this problem, and didn't tried to submit this bug, but now I know that is a bug in lolMiner, and I would switch to the another miner if this can't be resolved soon. Feel free to contact me for additional information if needed.

**Comment by Truster420** [comment]
I am having the same issue. Using windows 10 pro. Dual mining vrsc/beam. 4 cores/8 threads unused. Lolminer 1.31 randomly stops mining. Only message on cmd line "press any key to continue".

**Comment by cuakevinlex** [comment]
Same here,  i tried someone's solution by adding
` :START`
at the top

and adding 
```
Timeout 5
goto START
```

in the 
`:end`
